docs/content/preview/api/ysql/the-sql-language/statements/txn_begin.md

```diff
@@ -17,7 +17,11 @@ Use the `BEGIN` statement to start a transaction with the default (or specified)
 ## Syntax
 
 {{%ebnf%}}
-  begin
+  begin,
+  transaction_mode,
+  isolation_level,
+  read_write_mode,
+  deferrable_mode
 {{%/ebnf%}}
 
 ## Semantics
```

Comment:
Would be explicit and add something like the following under transaction_mode heading:

Supports isolation level, read write, and deferrable modes using the transaction mode syntax of `ISOLATION LEVEL`, `READ ONLY`, `READ WRITE`, and `[NOT] DEFERRABLE`.

For isolation level, you can specify Serializable, [...rest of paragraph]

Reply:
Added in https://github.com/yugabyte/yugabyte-db/pull/21782/commits/29d8edeabbf79725c438a0e9806eaa95c81619d8